contract_comm/gasprice_minimum/gasprice_minimum.go

```diff
@@ -137,12 +138,14 @@ func GetGasPriceMinimum(currency *common.Address, header *types.Header, state vm
 func UpdateGasPriceMinimum(header *types.Header, state vm.StateDB) (*big.Int, error) {
 	var updatedGasPriceMinimum *big.Int
 
+	gasLimit, _ := blockchain_parameters.GetBlockGasLimit(header, state)
```

Comment:
Ignoring an error is almost always forbidden. I would say it's reasonable to do here, but I'd add a comment.
```suggestion
	// If an error occurs, the default block gas limit will be returned and a log statement will be produced by contract_comm
	gasLimit, _ := blockchain_parameters.GetBlockGasLimit(header, state)
```